microsoft-edge/devtools-guide-chromium/network/reference.md

```diff
@@ -384,22 +496,82 @@ To hide the requests, turn off the **Hide data URLs** checkbox:
 ![The Hide Data URLs checkbox](./reference-images/hide-data-urls.png)
 
 
+<!-- ------------------------------ -->
+#### Hide extension URLs
+<!-- https://developer.chrome.com/docs/devtools/network/reference/#hide-extension-urls -->
+
+To focus on the code you author, you can filter out irrelevant requests sent by extensions you may have installed in Microsoft Edge.  Extension requests have URLs that start with `chrome-extension://`.
+
+To hide extension requests, in the **Filters** action bar, select the **More filters** dropdown menu, and then select **Hide extension URLs** so that a checkmark appears next to it.
+
+<!-- ![Extension URLs hidden from the Requests table](todo png: extension-urls-hidden.png, for Filter Requests) -->
+
+The status bar at the bottom displays the number of the shown requests out of the total.
+
+
+<!-- ------------------------------ -->
+#### Show only the requests with blocked response cookies
+<!-- https://developer.chrome.com/docs/devtools/network/reference/#show-blocked-cookies -->
+<!-- todo: clean up -->
+
+To filter out everything except the requests with response cookies blocked for any reason, in the **Filters** action bar, select the **More filters** dropdown menu, and then select **Blocked response cookies** so that a checkmark appears next to it.  <!--Try it on this [demo page](https://samesite-sandbox.glitch.me/). todo demo: create downstream demo for Filter Requests -->
+
+<!-- ![The Requests table shows only the requests with blocked response cookies](todo png: blocked-response-cookies.png, for Filter Requests) -->
+
+The status bar at the bottom displays the number of the shown requests out of the total.
+
+To find out the reason why a response cookie was blocked, select the request, open its **Cookies** tab, and hover over the **information** (**(i)**)<!-- todo: icon, for Filter Requests --> icon.
+
+Additionally, the **Network** tool shows a warning warning icon next to a request with cookies blocked either because of Chrome flags or configuration.  Hover over the icon to see a tooltip with a clue and click it to go to the Issues panel for more information.
```

Comment:
```suggestion
Additionally, the **Network** tool shows a warning warning icon next to a request with cookies blocked either because of Edge flags or configuration.  Hover over the icon to see a tooltip with a clue and click it to go to the Issues panel for more information.
```

Reply:
"warning warning" is a mid-sentence icon png to create